### PR TITLE
[tts] support browser

### PR DESCRIPTION
**Pull request by yulin-li**

### CODE REVIEW

File: src/sdk/Exports.ts [hunk 1]
```diff
@@ -69,3 +69,4 @@ export { SpeechSynthesizer } from "./SpeechSynthesizer";
 export { SpeechSynthesisResult } from "./SpeechSynthesisResult";
 export { SpeechSynthesisEventArgs } from "./SpeechSynthesisEventArgs";
 export { SpeechSynthesisWordBoundaryEventArgs} from "./SpeechSynthesisWordBoundaryEventArgs";
+export { SpeakerAudioDestination } from "../common.browser/SpeakerAudioDestination";
```
Comment (rhurey) [inline src/sdk/Exports.ts:72]: Objects that the SDK is going to expose should be in the SDK folder.

All the other folders are internal details.

File: src/sdk/Audio/AudioConfig.ts [hunk 2]
```diff
@@ -114,6 +114,27 @@ export abstract class AudioConfig {
...
+     * Creates an AudioConfig object representing the custom IPlayer object.
+     * You can use the IPlayer object to control pause, resume, etc.
+     * @member AudioConfig.fromSpeakerOutput
+     * @function
+     * @public
+     * @param {IPlayer} player - the IPlayer object for playback.
+     * @returns {AudioConfig} The audio output configuration being created.
+     * Added in version 1.12.0
+     */
+    public static fromSpeakerOutput(player?: IPlayer): AudioConfig {
```
Comment (rhurey) [inline src/sdk/Audio/AudioConfig.ts:127]: >IPlayer [](start = 45, length = 7)

Do we need to export this if it's a type parameter.

File: src/sdk/Exports.ts (hunk 1 above)
Comment (yulin-li) [reply src/sdk/Exports.ts]: Thanks, move the `SpeakerAudioDestination` to `sdk/Audio`

File: src/sdk/Audio/AudioConfig.ts (hunk 2 above)
Comment (yulin-li) [reply src/sdk/Audio/AudioConfig.ts:127]: I added some doc and export it.

**Pull request merged**